packages/table/src/table.tsx

```diff
@@ -789,6 +834,49 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         }
     }
 
+    private _handleFocusMoveLeft = (e: KeyboardEvent) => this.handleFocusMove(e, "left");
+    private _handleFocusMoveRight = (e: KeyboardEvent) => this.handleFocusMove(e, "right");
+    private _handleFocusMoveUp = (e: KeyboardEvent) => this.handleFocusMove(e, "up");
+    private _handleFocusMoveDown = (e: KeyboardEvent) => this.handleFocusMove(e, "down");
```

Comment:
no need for `_` prefix, we don't use this convention in the codebase because `private` is typically enough.